src/components/NavBar/style.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import { LinkColored } from '@/styles/text';
+
+export const NavBarContainer = styled.div`
+  /* top: 0;
```

Comment:
also, just catching another commented line that we may want to delete, going off alvaro!